virus/Mammalia_ventremi.md

```diff
@@ -0,0 +1,207 @@
+# The Mammalian Gut Associated Novel Obelisk Mammalia ventremi is Geographically Widespread and Contains an Efficient Encoding of Oblin-1, a Possible RNA Polymerase and RNA Interacting Protein.
+written by: [Teo Reid](https://github.com/reidmat5)
+
+### {Q1B Virus Name}
```

Comment:
Pleaes don't use the "Standard" titles, these are meant to tell you waht to change for your submission. Throughout the document this is true.